articles/tips-and-tricks-for-deep-learning/index.md

```diff
@@ -0,0 +1,252 @@
+
+### Data Processing
+
+#### Data augmentation
+
+This technique was first introduced in 2012 by the author of [AlexNet](https://papers.nips.cc/paper/2012/file/c399862d3b9d6b76c8436e924a68c45b-Paper.pdf/), Alex Krizhevsky. Data augmentation is used when the training data is not sufficient enough to learn a more generalizable model. This technique can be achieved by slightly altering the original image in various ways. 
+
+Let's take a look at pictorial examples of how the original image is altered.
+
+##### Original Image
+
+![An original image of a dog](/engineering-education/tips-and-tricks-for-deep-learning/original-image.jpg)<br>
+
+This image represents the original image which has not been altered in any way.
+
+##### Rotated Image
+
+![Rotated Image](/engineering-education/tips-and-tricks-for-deep-learning/rotate-image.jpg)<br>
+
+This image is slightly rotated to the right.
+
+##### Flipped Image
+
+![Flipped Image](/engineering-education/tips-and-tricks-for-deep-learning/flip-image.jpg)<br>
+
+##### Cropped Image 
+
+![Cropped Image](/engineering-education/tips-and-tricks-for-deep-learning/cropped-image.jpg)<br>
+
+This image is cropped. The focus has been put on one side of the image. 
+
+##### Noisy Image
+
+![Noisy Image](/engineering-education/tips-and-tricks-for-deep-learning/noisy-image.jpg)<br>
+
+This image is the version of the original image with added noise. 
+
+##### Color shifted Image
+
+![Color shifted Image](/engineering-education/tips-and-tricks-for-deep-learning/color-shifted-image.jpg)<br>
+
+##### Contrast changed Image
+
+![Contrast changed Image](/engineering-education/tips-and-tricks-for-deep-learning/contrast-changed-image.jpg)<br>
+
+If done correctly, data augmentation can be a very powerful tool. It forces the neural network to focus more on an image's attributes instead of the image themselves. Besides, it also increases the data needed for training.
+
+#### Batch Normalization (Batch Norm)
+
+Batch normalization is also known as Batch Norm. It is a technique used in deep learning to stabilize models and speed up learning.  
+
+Before we dive into how batch normalization works, let's first briefly discuss the working of a typical normalization technique. This discussion will make you better understand why we use batch normalization.
+
+Normalization is a pre-processing step on input data that helps us put data on a standard scale. This process is an important step as there might be variations in input data. Some input data might be having higher values while some, very low values. Let's consider an example of car mileages over a ten-year period to demonstrate non-normalized data. 
+
+One of the drivers, aged 50, has covered a mileage of 100,000 while another driver, aged 25, has only covered 1,000 miles on their cars. The mileage and age data from these two drivers vary widely thus might not fit on the same scale. 
```

Comment:
Rephrase the first sentence as, Consider a scenario where data related to two car drivers exists. One of them, aged 50, has covered 10000 while another, aged 25, has driven 1000 miles. 

Also, mileage might not be the right term here. Mileage refers to Km/Litre of fuel.